### PR TITLE
Notify compliance 48

### PR DESCRIPTION
**Pull request by terrazoon**
## Description

The original fix to compliance #46 was very empirical and ad hoc, based on errors we were seeing in the logs.  Based on further analysis we only need to do one test.

## Security Considerations

Same as compliance #46

### CODE REVIEW

File: app/notify_client/__init__.py [hunk 1]
```diff
@@ -64,14 +67,15 @@ def check_inactive_user(self, *args):
...
-            ):
+            if self.is_calling_signin_url(arg):
+                # if (
+                #     "get-login-gov-user" in arg
+                #     or "user/email" in arg
+                #     or "/activate" in arg
+                #     or "/email-code" in arg
+                #     or "/verify/code" in arg
+                #     or "/user" in arg
+                # ):
```
Comment (ccostino) [inline app/notify_client/__init__.py:78]: Just double checking that we're sure we don't need to test all of these URL paths?  How was this confirmed?

And if that's the case, I think it's safe to remove the commented out code then to help clean things up!

**Review by ccostino — COMMENTED** [review]
Thanks, @terrazoon!  I had just one quick question to verify the changes to the test?

File: app/notify_client/__init__.py (hunk 1 above)
Comment (terrazoon) [reply app/notify_client/__init__.py]: I did grep -ri "/email-code" ./**/**.py and verified that the uses of that partial path all start with "/user", then I checked the others.

I guess we could test invites and 90 day email verifications again

Comment (ccostino) [reply app/notify_client/__init__.py]: Yes, we should make sure the invites and 90 day email verification work still with this.  We should be able to test in staging by modifying our user records in the DB to trigger the email verification again.  Thanks!

**Review by xlorepdarkhelm — APPROVED** [review]
LGTM

**Comment by ccostino** [comment]
@terrazoon this just needs a quick update with `main` again and it should be good to go.  Thanks!

**Review by ccostino — APPROVED** [review]
Thanks, @terrazoon!

**Pull request merged**